docs/dev-guide/iframe.md

```diff
@@ -253,27 +313,35 @@ api.setVideoInputDevice(deviceLabel, deviceId);
 ```javascript
 api.startRecording(options);
 ```
-
 * **stopRecording** - Stops an ongoing file recording or streaming session.
 
 ```javascript
 api.startRecording(mode);
 ```
 
-You can control the embedded Jitsi Meet conference by calling `executeCommand` on the `JitsiMeetExternalAPI` object:
+You can control the embedded Jitsi Meet conference by calling **`executeCommand`** on the **`JitsiMeetExternalAPI`** object:
 
 ```javascript
 api.executeCommand(command, ...arguments);
 ```
 
-The `command` parameter is a string containing the name of the command. The following commands are currently supported:
+## **Commands**
+
+The command parameter is a string which contains the command name. 
+
+The following commands are supported:
+
+* **displayName** - Sets the display name of the local participant. 
+
+This command requires one argument to set the new display name.
 
-* **displayName** - Sets the display name of the local participant. This command requires one argument - the new display name to be set.
 ```javascript
 api.executeCommand('displayName', 'New Nickname');
 ```
+* **password** - Sets the password for the room. 
+
+Set the password name as the command's single argument.
```

Comment:
I don't think the extra explanation is necessary here.

Reply:
removed